pkg/apis/seedmanagement/v1alpha1/zz_generated.conversion.go

```diff
@@ -286,8 +296,7 @@ func autoConvert_v1alpha1_ManagedSeedSpec_To_seedmanagement_ManagedSeedSpec(in *
 	if in.SeedTemplate != nil {
 		in, out := &in.SeedTemplate, &out.SeedTemplate
 		*out = new(core.SeedTemplate)
-		// TODO: Inefficient conversion - can we improve it?
-		if err := s.Convert(*in, *out, 0); err != nil {
```

Comment:
Due to this, I had to introduce https://github.com/gardener/gardener/pull/3651/files#diff-c7b64578d2ac95623dfd44edeb4a58700b755fa9bda6b8267965ec156363a6e6R76-R83. I'm not sure why the generation does not find the functions here: https://github.com/gardener/gardener/blob/6f6cf8a41b46e01bac9761c01d0f8bc0e22e8c61/pkg/apis/core/v1beta1/zz_generated.conversion.go#L4366-L4369

I played with `--extra-peer-dirs` but without success. Any pointers are appreciated.

/cc @stoyanr @ialidzhikov

Reply:
We also ran into this problem and it looks like a regression in the code generator for k8s 1.20: https://github.com/kubernetes/kubernetes/issues/98380

Reply:
Thanks @schrodit for pointing this out. Then I guess the workaround I applied is fine for now until a fix was released, right?